src/antsibull/new_ansible.py

```diff
@@ -91,13 +92,24 @@ def new_ansible_command():
         os.path.join(app_ctx.extra['data_dir'], app_ctx.extra['pieces_file']))
     dependencies = asyncio.run(get_version_info(collections, app_ctx.pypi_url))
 
-    ansible_core_version = dependencies.pop('_ansible_core')[0]
+    ansible_core_release_infos = dependencies.pop('_ansible_core')
+    ansible_core_versions = [
+        (PypiVer(version), data[0]['requires_python'])
+        for version, data in ansible_core_release_infos.items()
+    ]
+    ansible_core_versions.sort(reverse=True, key=lambda tuple: tuple[0])
```

Comment:
Using `tuple` as a variable name is a bad idea.

Reply:
Fixed in aeeae518996c190f62311ebd8c5b665c5d8e0baa.